### PR TITLE
Update new-theme webpack to v4.41 and its dependencies

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | We wants to update webpack on our themes so we will be able to merge them into only one package.json, I updated webpack to v4.41 and its linked dependencies. Some minors changes had to be done to our files, this PR is aiming the building of assets of migrated pages. I also kicked chunks from prod build command so it avoid too much noises for peoples wanting to build production assets
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | What you need to test is the fact that migrated pages are the same as before this PR, nothing should change after building the assets of admin-dev/theme/new-theme !

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17426)

### CODE REVIEW

**Review by eternoendless — DISMISSED** [review]

File: admin-dev/themes/new-theme/package.json [hunk 1]
```diff
@@ -18,37 +18,46 @@
...
+    "vue": "^2.6.11",
+    "vue-loader": "^15",
     "vue-resource": "^1.5.1",
     "vue-router": "^2.8.1",
     "vue-i18n": "^8.15.1",
-    "vue-template-compiler": "^2.6.7",
-    "vuex": "^2.5.0"
+    "vue-template-compiler": "^2.6.11",
+    "vuex": "^2.5.0",
+    "webpack": "^4.41",
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/package.json]: webpack should be in devDependencies :thinking:

File: admin-dev/themes/new-theme/package.json [hunk 2]
```diff
@@ -18,37 +18,46 @@
     "bootstrap-tokenfield": "^0.12.0",
     "bourbon": "^4.2.6",
     "compass": "^0.1.1",
+    "copy-webpack-plugin": "^5.1.1",
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/package.json]: same for this one, in devDependencies (and below too)

File: admin-dev/themes/new-theme/package.json (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/package.json]: Then why eslint cry about it when it's in dev dep?

File: admin-dev/themes/new-theme/package.json (hunk 2 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/package.json]: Same as my upper comment

File: admin-dev/themes/new-theme/package.json [hunk 3]
```diff
@@ -18,37 +18,46 @@
...
     "fos-routing": "0.0.6",
     "jets": "0.11.0",
     "jquery-ui-dist": "^1.12.1",
     "jwerty": "^0.3.2",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
     "lodash.escaperegexp": "^4.1.2",
     "magnific-popup": "^1.1.0",
-    "mobile-detect": "^1.4.3",
+    "mini-css-extract-plugin": "^0.8.0",
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/package.json]: same

**Review by PierreRambaud — DISMISSED** [review]
Require changes

**Review by PierreRambaud — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]
LGTM

**Comment by Progi1984** [comment]
@NeOMakinG Please fix the ESLint check

**Review by PierreRambaud — DISMISSED** [review]

File: admin-dev/themes/new-theme/package.json [hunk 4]
```diff
@@ -57,33 +57,35 @@
...
+    "style-loader": "^1.1.3",
     "uglify-save-license": "^0.4.1",
-    "uglifyjs-webpack-plugin": "^1.3.0",
-    "vue-loader": "^11",
+    "uglifyjs-webpack-plugin": "^2.2.0",
+    "vue-loader": "^15",
     "vue-style-loader": "^4.1.2",
-    "webpack": "^2.7"
+    "webpack": "^4.41",
+    "webpack-cli": "^3.3.10",
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/package.json:90]: @NeOMakinG Have you checked license of `webpack-cli` & `webpack-fix-style-only-entries` ?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/package.json:90]: MIT and ISC @Progi1984

Comment (Progi1984) [reply admin-dev/themes/new-theme/package.json:90]: Is it ok for you @ttoine ?

Comment (ttoine) [reply admin-dev/themes/new-theme/package.json:90]: I confirm that MIT and ISC are permissive licences compatible with the PrestaShop project's OSL and AFL license.
May I ask if this is for the core, or for the Classic theme?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/package.json:90]: Core new-theme

**Review by Progi1984 — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by eternoendless — DISMISSED** [review]
LGTM

**Review by matks — DISMISSED** [review]

**Comment by SD1982** [comment]
Hello @NeOMakinG  it seems that with this PR and on the migrated page the lateral menu has became scrollable until he disappear : 

**With PR on migrated page**
https://drive.google.com/file/d/1OWpqN0A8omk_cIMEf0Yt8Z71aBXvRPFx/view?usp=sharing

**With PR on non-migrated page**
https://drive.google.com/file/d/1I9zFue1oFD8odjkJztn1op7r18nuxt5y/view?usp=sharing

**Without PR on any page migrated or not**
https://drive.google.com/file/d/1fxEsT3Zfo_799J_Snjq6yVV2rTjmBFJ1/view?usp=sharing

**Comment by NeOMakinG** [comment]
Any ideas @PrestaShop/prestashop-core-developers @PrestaShop/prestashop-qa ? I can't reproduce this myself

**Comment by SD1982** [comment]
Hi @khouloudbelguith can your try if you can reproduce the behaviour that i have noticed ?

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

Yes, I have the same issue reproduced by @SD1982.
You need to navigate to the BO => Advanced Parameters => Information page.
I attached a screen record
[Performance.webm.zip](https://github.com/PrestaShop/PrestaShop/files/4355643/Performance.webm.zip)
I used:
1. Chrome Version 77.0.3865.90 (Build officiel) (64 bits)
2. PHP version 7.2.23

Thanks!

**Review by Progi1984 — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by SD1982** [comment]
Hi @NeOMakinG i have noticed this in International => localization => currencies => Add new

**Before PR** 
currencies iso before

**After PR**
currencies iso after

**Comment by NeOMakinG** [comment]
It's fine on `develop` and on this branch, the case is grey when unchecked and white when checked.

Doesn't look linked to this PR

**Review by matks — APPROVED** [review]

**Comment by SD1982** [comment]
Hi @NeOMakinG Thanks for the information I have retested with a fresh install and i don't reproduce the behaviour I have noticed before I think it was a cache navigator issue . Thanks @NeOMakinG

**Comment by eternoendless** [comment]
Thank you @NeOMakinG

**Pull request merged**